modules/account-lib/src/coin/hbar/transactionBuilder.ts

```diff
@@ -99,6 +102,18 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     return this;
   }
 
+  /**
+   * Set an external transaction signature
+   *
+   * @param signature Hex encoded signature string
+   * @param keyPair The public key keypair that was used to create the signature
+   * @returns This transaction builder
+   */
+  signature(signature: string, keyPair: KeyPair): this {
```

Comment:
could we make this `addSignature`

Reply:
We are not adding verbs into the builder methods names. `signature` adheres to that convention.